canvasapi/account.py

```diff
@@ -1702,14 +1804,66 @@ def update_role(self, role, **kwargs):
 @python_2_unicode_compatible
 class AccountNotification(CanvasObject):
     def __str__(self):  # pragma: no cover
-        return "{}".format(self.subject)
+        return "{} ({})".format(self.subject, self.id)
+
+    def update_global_notification(self, account_notification, **kwargs):
+        """
+        Updates a global notification.
+
+        :calls: `PUT /api/v1/accounts/:account_id/account_notifications/:id \
+        <https://canvas.instructure.com/doc/api/account_notifications.html#method.account_notifications.update>`_
+
+        :param account_notification: The notification to update with.
+        :type account_notification: dict
+
+        :rtype: :class:`canvasapi.account.AccountNotification`
+        """
+        required_key_list = ["subject", "message", "start_at", "end_at"]
+        required_keys_present = all(
+            (x in account_notification for x in required_key_list)
```

Comment:
Please remove the extra pair of parentheses.

Reply:
Removed.